src/main/java/org/apache/cassandra/sidecar/cdc/output/Output.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.cassandra.sidecar.cdc.output;
+
+import java.io.Closeable;
+import org.apache.cassandra.sidecar.cdc.Change;
+
+/**
+ * Interface for emitting Cassandra PartitionUpdates
+ */
+public interface Output extends Closeable
+{
+    void emitChange(Change change) throws Exception;
```

Comment:
besides , I want to add some method such emitChange but the return is partitionUpdate

Reply:
How the method signature would look like in that case?